src/site/stages/build/process-cms-exports/schemas/transformed/paragraph-table.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  type: 'object',
+  properties: {
+    entity: {
+      type: 'object',
+      required: ['fieldTable'],
+      fieldTable: {
+        type: 'object',
+        required: ['tableValue', 'value'],
+        properties: {
+          tableValue: {
+            type: 'string',
+          },
+          value: {
+            type: 'array',
```

Comment:
Hrm. So this is actually an "array-like" object in `pages.json`. Because it also has a `caption`. Not sure if that `caption` ever has content, but...it's there.